repose-aggregator/components/filters/client-auth/src/test/java/org/openrepose/filters/clientauth/openstack/OpenStackAuthenticationHandlerTest.java

```diff
@@ -242,7 +243,7 @@ public void standUp() throws Exception {
         }
 
         @Test
-        public void tenantIdFromTokenMatchesURI() {
+        public void tenantIdFromTokenMatchesURI() throws TimeoutException {
```

Comment:
it's odd to me that theses tests didn't already say throws exception, that's what the intellij template will do, so you don't have to worry about this sort of thing. le sigh.